docker/k8s/vtadmin/Dockerfile

```diff
@@ -47,9 +53,15 @@ RUN deluser node && \
    chown -R vitess:vitess /vt && \
    chown -R vitess:vitess /vtdataroot
 
-USER vitess
+FROM nginx:latest AS nginx
+
+COPY --from=node /vt/web/vtadmin/build /var/www/
+COPY --from=node nginx.conf /etc/nginx/nginx.conf
+COPY --from=node entrypoint.sh /entrypoint.sh
+
```

Comment:
we should specify a user (vitess) to not run the container as root

Reply:
Going to specify the user as nginx in this case since it's available. I'll give it the necessary permissions